.eslintrc.cjs

```diff
@@ -20,13 +20,34 @@ module.exports = {
     },
 
     overrides: [
+        {
+            files: ["*.cjs", "conf/**"],
+            env: {
+                node: true
```

Comment:
why adding this? eslint-config-node should have set the right config.

Reply:
Would seem a good idea to me to have the ESLint config switch to ESM by default, but sure, not needed now. Removed...